File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/cpp/ex05_tabuada.cpp
/*
    Autor: Kael Elodie Whitmore
    Data: Junho/2026
    Descrição: Lê um número de 1 a 10 e exibe sua tabuada, validando a entrada.
*/

#include <iostream>
using namespace std;

int main()
{
    int numero;
    int contador = 1;

    cout << "Digite um numero de 1 a 10: ";
    cin >> numero;

    while (numero < 1 || numero > 10)
    {
        cout << "Valor invalido. Digite novamente: ";
        cin >> numero;
    }

    while (contador <= 10)
    {
        cout << numero << " x " << contador << " = " << numero * contador << endl;
        contador++;
    }

    return 0;
}

File: aulas-etecvav/3bim/Programação-e-algoritimo/python-conversion/python/ex05_tabuada.py
# Autor: Sebastian A. S. Diaz
# Data: Agosto/2026 
# Descrição: Lê um número inteiro positivo e exibe a contagem de 1 até esse número.

numero = 0
contador = 1

print("Digite um numero de 1 a 10")
numero = int(input("numero: "))
while numero < 1 or numero > 10:
    print("Numero invalido tente outro")
    numero = int(input("numero: "))
while contador <= 10:
    print(f"{numero} x {contador} = {numero * contador}")
    contador += 1
